src/components/MenuPanel/MenuPanel.js

```diff
@@ -200,19 +207,20 @@ class AnimatedMenuPanel extends React.Component {
 
   render() {
     const { animate } = this.state
-    const { opened, autoClosing, onCloseMenuPanel, ...props } = this.props
+    const { openProgress, autoClosing, onCloseMenuPanel, ...props } = this.props
```

Comment:
`autoClosing` doesn't seem to be used in `render()` (although maybe it should control some of this behaviour cc @bpierre?)

Reply:
I had not realized that the lates `animate` changes removed `autoClosing` from render, will update.